uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -211,6 +234,33 @@ define([
       }
     }
 
+    /**
+     * Returns options for an action confirmation dialog
+     *
+     * @param  {String} action
+     * @return {Object}
+     */
+    function getConfirmationDialogOptions (action) {
+      var dialogParams = actions[action].dialog;
+
+      return {
+        title: 'Confirm ' + dialogParams.title + '?',
+        copyCancel: 'Cancel',
```

Comment:
this should be a default value.

Reply:
I would not change default values since maybe somewhere it is expected that this value is empty.